bahmni-commons-api/src/main/java/org/bahmni/module/bahmnicommons/util/SqlQueryHelper.java

```diff
@@ -0,0 +1,134 @@
+package org.bahmni.module.bahmnicommons.util;
+
+import org.apache.commons.lang3.StringUtils;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.bahmni.module.bahmnicommons.model.searchParams.AdditionalSearchParam;
+import org.codehaus.jackson.map.ObjectMapper;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SqlQueryHelper {
```

Comment:
This class maybe redundant. After you have resolved and removed other redudant classes as indicated above. check whether this is used at all